src/wordCount/Main.java

```diff
@@ -0,0 +1,90 @@
+		unText=unText.replaceAll("[[\\.\\?\\!\\,\\;\\:\\{\\}\\(\\)\\']]", "");
+		unText=unText.toLowerCase();
+		String[] wordsArr = unText.split(" +");
+		String[] testArr = new String[]{"the","and","social","rights","freedoms"};
+
+		HashMap<String,Integer> wordHash = new HashMap<String,Integer>();
+
+		for (String word : wordsArr)
+		{
+			if(wordHash.containsKey(word))
+			{
+				wordHash.put(word,wordHash.get(word)+1);
+				//System.out.println(word+" "+wordHash.get(word));
+			}else
+			{
+				wordHash.put(word,1);
+				//System.out.println(word+" 1");
+			}
+
+		}
+
+//Sorted Words
+		ArrayList<HashMap.Entry<String,Integer>> sortedWords = new ArrayList<HashMap.Entry<String,Integer>>();
+		sortedWords.addAll(wordHash.entrySet());
+
+		Collections.sort(sortedWords,new Comparator<Map.Entry<String,Integer>>()
+			{
+				public int compare(HashMap.Entry<String,Integer> o1,HashMap.Entry<String,Integer> o2)
+				{
+					return o2.getValue().compareTo(o1.getValue());
+				}
+			}
+		);
```

Comment:
So i looked up compareTo(), i'm not sure it's for integers. but if it works it works! Another way i saw people doing this is like so: 
```return o2.getValue() - (o1.getValue()); ```